examples/out_lib/out_lib.c

```diff
@@ -30,13 +32,45 @@ int my_stdout_json(void *record, size_t size, void *data)
     return 0;
 }
 
-int my_stdout_msgpack(void* data, size_t size)
+int my_stdout_msgpack(void *record, size_t size, void *data)
 {
     printf("[%s]",__FUNCTION__);
-    msgpack_object_print(stdout, *(msgpack_object*)data);
-    printf("\n");
 
-    flb_lib_free(data);
+    msgpack_unpacked result;
+    size_t off = 0, cnt = 0;
+    struct flb_time tmp_time;
+    msgpack_object *x;
+
+    msgpack_unpacked_init(&result);
+
+    while (msgpack_unpack_next(&result, record, size, &off) == MSGPACK_UNPACK_SUCCESS) {
+        printf("[%zd]: [", cnt++);
+        flb_time_pop_from_msgpack(&tmp_time, &result, &x);
```

Comment:
Could you check a type of result.data before `flb_time_pop_from_msgpack` ?
We expect the type is MSGPACK_OBJECT_ARRAY and its size is 2.
https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1#entry